AMP-91621 update Profile API dev doc on multiple computation IDs support (#323)
amplitude/amplitude-dev-center: docs/analytics/apis/user-profile-api.md

```

## Get single/multiple computation

!!!note "Feature availability"

    This feature is available in accounts with Amplitude Audiences.

> Retrieves a single or multiple computation by ID(s). Find the computation ID by navigating to the computation in the Audiences web app and copying the ID at the end of the URL. The ID is bold in this example: https://app.amplitude.com/audiences/org_name_00000/computations/**t14bqib**/

Reviewer: ```suggestion
Retrieves a single or multiple computation by ID. Navigate to the computation in Audiences to find and copy the ID at the end of the URL. For example, `t14bqib` is the ID in  `https://app.amplitude.com/audiences/org_name_00000/computations/t14bqib/`
```
